deploy/stack/compose/setup.sh

```diff
@@ -87,19 +57,12 @@ tweak_bash_interactive_settings() {
 }
 
 main() {
-	local host_ip=$1
-	local worker_ip=$2
-	local worker_mac=$3
-	local compose_dir=$4
-
 	update_apt
 	install_docker
 	install_kubectl
 
-	# setup_layer2_network "$host_ip"
-
-	setup_compose_env_overrides "$host_ip" "$worker_ip" "$worker_mac" "$compose_dir"
-	docker compose --env-file "$compose_dir"/.env -f "$compose_dir"/docker-compose.yml up -d
+	local compose_dir="/sandbox/stack/compose"
+	docker-compose -f "$compose_dir"/docker-compose.yml up -d
```

Comment:
im curious why the move to the older `docker-compose` command over the newer `docker compose` command?

Reply:
It's just an old, bad habit of mine. I'll fix it!